tests/integration/olp-cpp-sdk-dataservice-read/VersionedLayerClientTest.cpp

```diff
@@ -247,6 +248,41 @@ class DataserviceReadVersionedLayerClientTest : public ::testing::Test {
                                    olp::http::HttpStatusCode::OK),
                                HTTP_RESPONSE_BLOB_DATA_PREFETCH_6));
 
+    ON_CALL(*network_mock_, Send(IsGetRequest(URL_BLOB_DATA_269), _, _, _, _))
+        .WillByDefault(
+            ReturnHttpResponse(olp::http::NetworkResponse().WithStatus(200),
```

Comment:
```suggestion
            ReturnHttpResponse(olp::http::NetworkResponse().WithStatus( olp::http::HttpStatusCode::OK),
```
Also in other places.